chroot-script

```diff
@@ -104,8 +104,8 @@ chrootmirror() {
 
   # add security.debian.org:
   case "$RELEASE" in
-    unstable|sid) ;;  # no security pool available
-    jessie|stretch|buster)
+    unstable|sid|stretch) ;;  # no security pool available
+    jessie|buster)
```

Comment:
are you sure this change makes sense? stretch is in between jessie and buster (see https://wiki.debian.org/DebianReleases)?

Reply:
Unfortunately, I am sure. The repository is defunct but not worth reporting upstream to Debian because that suite is end of life.

Reply:
In theory, I agree. It should also be affected. But in practice, it's not as I could see from the CI tests. I don't have an explanation why it is as is. I just noticed that oddly 1 Debian security repository for an EOL suite is broken.

Reply:
Porting to `archive.debian.org` might fix this issue but lets consider that a separate issue that is not worth doing for an EOL suite? :)

Reply:
Not that not enabling security repository for stretch didn't reduce the security or change installed packages.

* Previously: the stretch build would just show an error message that the stretch repository is unavailable, ignore it, and proceed.
* After this PR that enabled strict error checking: That resulted in a build failure so we at least know something is wrong. It only made this issue apparent. But the issue was there all along even if nobody noticed.This PR however didn't worsen anything. Well, could argue in theory if that stretch security repository ever comes back online then it's missing from these builds. If that is considered an issue then indeed code to make stretch use `archive.debian.org` might be required.